packages/FormElement/src/components/ExtraPanel/ExtraPanel.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import extraPanelStyles from "./ExtraPanel.styles";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+function ExtraPanel(props) {
```

Comment:
`ExtraPanel` isn't sitting right with me.  I'm not sure what name would best describe this, especially in contrast to the `Description` which is basically the same thing, but in a different layout position... 🤔 

How about `Instructions`? Would that be suggestive if its actual purpose? A user would want to read instructions before proving input, right?

cc @krismckinnonacl

Reply:
I'm figured we would want something agnostic rather than specific? I'm fine with instructions anyway.

Reply:
Going with Instructions for now